server/market/market.go

```diff
@@ -327,6 +425,20 @@ func (m *Market) processOrder(rec *orderRecord, epoch *EpochQueue, errChan chan<
 		return
 	}
 
+	// Ensure that the received order does not use locked coins.
+	lockedCoins := m.coinsLocked(ord)
```

Comment:
These coins are being checked in both the order router and the market. Is that the intention?

Reply:
Since the epoch processing loop in `Market.runEpochs` runs concurrently with the order router handlers registered with the `AuthManager`, and coins aren't locked until `runEpochs` does `Market.processOrder`, there needs to be a check somewhere in the synchronous processing pipeline of `runEpochs`/`processOrder`.  I think we could eliminate `(*OrderRouter).checkPrefixTrade`, but the early extra check seemed fine.